styx-flyte-client/src/main/proto/flyteidl/admin/common.proto

```diff
@@ -0,0 +1,214 @@
+syntax = "proto3";
```

Comment:
It might be useful that we put a README.md under this folder stating where we copied these protos.